app/destination/page.js

```diff
@@ -72,19 +71,30 @@ export const Destinations = () => {
       thumbnail: "/destination/image-titan.png",
     },
   ];
-  const onAddOrRemovePlanet = (name, index) => {
-    // TASK - React 1 week 2
-    if (selectedPlanets.includes(name)) {
-      onAddPlanet(selectedPlanets.filter((planet) => planet !== name));
-    } else {
-      onAddPlanet([...selectedPlanets, name]);
-    }
 
+  const onAddOrRemovePlanet = (index) => {
```

Comment:
I'm not sure if I fully understand this, but to me it looks like this add/remove functionality could be written in a simpler way: 

1. Let's have two different buttons and two different functions, one for add and one for remove.
2. It looks like `onAddPlanet` is called even in case of removing a planet?
3. The prop `onAddPlanet` should take as minimum information as possible, ideally the index of the added planet. It should not accept a function as argument as that complicates the matter, unless it is absolutely necessary. So I encourage you to rethink this structure and make it simpler, i.e., see _why_ this data needs to be sent to the parent and try to minimize that data. 

If in doubt, I suggest reading this page: https://react.dev/learn/thinking-in-react